zkevm-circuits/src/sig_circuit/utils.rs

```diff
@@ -7,12 +7,16 @@ use halo2_ecc::{
 };
 use halo2_proofs::{
     circuit::Value,
-    halo2curves::secp256k1::{Fp, Fq},
+    halo2curves::secp256k1::{Fp as Fp_K1, Fq as Fq_K1},
+    halo2curves::secp256r1::{Fp as Fp_R1, Fq as Fq_R1},
 };
 
 // Hard coded parameters.
 // FIXME: allow for a configurable param.
-pub(super) const MAX_NUM_SIG: usize = 128;
+pub(super) const MAX_NUM_SIG_K1: usize = 128;
+// MAX_NUM_SIG/4 for MAX_NUM_SIG_R1 ?
+pub(super) const MAX_NUM_SIG_R1: usize = 32;
```

Comment:
change these 2 values to 110 and 15.

Reply:
updated in [78d85e8](https://github.com/scroll-tech/zkevm-circuits/pull/1351/commits/78d85e82537accb6d0352043a08f2029e1b0c771)